packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -8,6 +20,16 @@
 
 - Add support for invoking function using payload contents ([PR #3472](https://github.com/forcedotcom/salesforcedx-vscode/pull/3472))
 
+- Bump SDR to 3.0.0 ([PR #3497](https://github.com/forcedotcom/salesforcedx-vscode/pull/3451))
```

Comment:
I might have an outdated develop branch. These entries should be in the previous changelog